### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by rejhgadellaabsl** [cdonohue/clean-ui#3]
I tried installing the theme:

`apm install clean-ui`

and I get this error:

> Unable to download https://www.atom.io/api/packages/clean-ui/versions/1.8.2/tarball: 400 Bad Request Repository inaccessible

Then I came here (github) to file an issue and noticed the message saying we should use prestine-ui. Maybe update the listing on atom theme page as well?

**Comment by pbraga** [comment]
+1

**Comment by cdonohue** [comment]
I would love to update the them listing on the Atom theme page, but there is no way to push updates (see below)

When the fork was removed (`atom-one-dark`), the link between GitHub and APM was lost and no further updates could be pushed to `clean-ui`.

So, all of the updates now have to go to another named theme. I have filed and issue with the `apm` team here.
> https://github.com/atom/apm/issues/644

Sorry for the confusion, but `pristine-ui` is one that I have control over. `clean-ui` can only be modified by the `apm` team at this point.
